.changes/next-release/bugfix-AWSSDKforJavav2-7ab9c30.json

```diff
@@ -0,0 +1,6 @@
+{
+    "type": "bugfix",
+    "category": "AWS SDK for Java v2",
+    "contributor": "",
+    "description": "Bytes in a toJSon of EnhancedDoucment  should be represented in a hexadecimal encoded format for their binary representation."
```

Comment:
Nit: toJSon -> toJson, EnhancedDoucment -> EnhancedDocument, '  ' -> ' ' (looks like double space after EnhancedDoucment)

- Why does it say that it should be represented as a `hexadecimal encoded format`? 
- Can you add a motivation why the chosen representation is the right one?

Reply:
Updated change log.
`hexadecimal encoded format` this is incorrect it should have been base64  ,  I corrected the changeog.
[DynamoDB Item sizes and formats](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/CapacityUnitCalculations.html) specifies 

```
A binary value must be encoded in base64 format before it can be sent to DynamoDB, 
```
And this is how it is shown in web console.